cli/src/base/move_package_resolver.rs

```diff
@@ -0,0 +1,200 @@
+use std::{collections::HashMap, fs};
+
+use anyhow::{anyhow, bail, Context, Result};
+use reqwest::blocking::Client;
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use toml_edit::easy::{self, value::Table, Value};
+use utils::{env::MOVE_HOME, movey_credential};
+
+#[derive(Deserialize, Debug)]
+pub struct Dependencies {
+    pub dependencies: MoveyDependencies,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct MoveyDependencies {
+    pub movey: MoveyResolver,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct MoveyResolver {
+    pub resolver: String,
+    pub packages: Option<HashMap<String, String>>,
+}
+
+#[derive(Debug, Serialize)]
+pub struct MovePackageResolver {
+    deps_map: HashMap<String, String>,
+    deps_resolved: HashMap<String, MoveDependency>,
+}
+
+impl MovePackageResolver {
+    fn new() -> Self {
+        MovePackageResolver {
+            deps_map: HashMap::new(),
+            deps_resolved: HashMap::new(),
```

Comment:
How about we rename `deps_map` --> `deps`, so that it is consistent with the field below `deps_resolved` (without suffix 'map')